Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Properties.cs

```diff
@@ -0,0 +1,171 @@
+                }
+            }
+        }
+
+        private static void OnAspectRatioChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateAspectRatio();
+        }
+
+        private static void OnCircularCropChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateAspectRatio();
+            target.UpdateThumbsVisibility();
+            target.UpdateMaskArea();
+        }
+
+        private static void OnIsSecondaryControlButtonVisibleChanged(
+            DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var target = (ImageCropper)d;
+            target.UpdateThumbsVisibility();
+        }
+
+        /// <summary>
+        ///  Gets or sets the source of the cropped image.
+        /// </summary>
+        public WriteableBitmap Source
+        {
+            get { return (WriteableBitmap)GetValue(SourceProperty); }
+            set { SetValue(SourceProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the aspect ratio of the cropped image，the default value is -1.
+        /// </summary>
+        public double AspectRatio
+        {
+            get { return (double)GetValue(AspectRatioProperty); }
+            set { SetValue(AspectRatioProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets whether to use a circular ImageCropper.
+        /// </summary>
+        public bool CircularCrop
+        {
+            get { return (bool)GetValue(CircularCropProperty); }
+            set { SetValue(CircularCropProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the mask on the cropped image.
+        /// </summary>
+        public Brush Mask
+        {
+            get { return (Brush)GetValue(MaskProperty); }
+            set { SetValue(MaskProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value for the style to use for the primary thumbs of the ImageCropper.
+        /// </summary>
+        public Style PrimaryThumbStyle
+        {
+            get { return (Style)GetValue(PrimaryThumbStyleProperty); }
+            set { SetValue(PrimaryThumbStyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value for the style to use for the secondary thumbs of the ImageCropper.
+        /// </summary>
+        public Style SecondaryThumbStyle
+        {
+            get { return (Style)GetValue(SecondaryThumbStyleProperty); }
+            set { SetValue(SecondaryThumbStyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether secondary thumbs is displayed.
+        /// </summary>
+        public bool IsSecondaryThumbVisible
```

Comment:
How about an enum for which to show
`public enum ThumbPlacement {  All, Corners, TopLeftBottomRight, TopRightBottomLeft }`
Or maybe `DiagonalLeft` and `DiagonalRight` for the last options

Reply:
It could be flagged so they could be used in combination?  Though I'm trying to think if UWP supports setting multiple flag values to an enum by default or if CreateFromString could be used on an enum property...

Reply:
I can't think of more than four possible options. If someone wants only top thumbs then they're just weird...

Reply:
To start with I think two options would be fine `All`, and `Corners`. Could even do `None`

Reply:
I agree. That's a great idea!